docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -2963,18 +2963,22 @@ column-definition is:
 <<<
 ```
 data-type is:
-    char[acter] [(length [characters])]
-          [character set char-set-name]
-          [upshift] [[not]casespecific]
+    char[acter] [length [characters]]

```

Comment:
I think you meant, "char[acter] (length [characters])"  (That is, the length is surrounded by parentheses, and it is not optional.)

Reply:
Thanks for your review @DaveBirdsall 

`length [characters]` is optional. 
If `length` is not specified, the default is 1.
![default](https://user-images.githubusercontent.com/20532956/42791889-58a3b3a0-89a5-11e8-9d65-2ead077e0ab2.png)

Reply:
Ah, good catch! I stand corrected. So, the original text, "char[acter] [(length [characters])] was correct? (The parentheses are needed if a length is supplied.)

Reply:
FYI: I double-checked the parser code. Length is optional for CHAR and NCHAR. Length is required for VARCHAR and NCHAR VARYING.

Reply:
Yes, the origin text is right, I've corrected my writing 😁